centreon/src/Core/Security/Authentication/Domain/Provider/OpenIdProvider.php

```diff
@@ -1045,9 +1023,9 @@ private function logAuthenticationDebug(string $message, array $content): void
      */
     private function logAuthenticationInfo(string $message, ?array $content = null): void
     {
-        $this->centreonLog->insertLog(
-            CentreonUserLog::TYPE_LOGIN,
-            "[openid] [INFO] {$message}" . ($content !== null ? ' : ' . json_encode($content) : '')
+        CentreonLog::create()->info(
+            CentreonLog::TYPE_LOGIN,
+            "[Openid] {$message}" . ($content !== null ? ' : ' . json_encode($content) : '')
```

Comment:
json_encode can be on error, you need to use the [JSON_THROW_ON_ERROR](https://www.php.net/manual/en/json.constants.php#constant.json-throw-on-error) flag to have an improved error handling ;)

Reply:
$content should be passed in argument of error() for the custom options

Reply:
ok